### DISCUSSION TITLE
Error: 400 '$.stop' is invalid

### DISCUSSION THREAD

**Discussion by NoahDragon** [danny-avila/LibreChat#1163] [Troubleshooting] [answered]
While using the gpt4-vision-preview, it throws the following error:

```
Something went wrong. Here's the specific error message we encountered: Error: 400 '$.stop' is invalid. Please check the API reference: https://platform.openai.com/docs/api-reference.
```

Wondering is this some error relate to configurations?

![image](https://github.com/danny-avila/LibreChat/assets/5704064/08be0e15-485b-4e06-aa49-85688eeea75d)

**Comment by danny-avila** [comment, marked as answer]
It’s not supported yet. But will be soon!

**Reply by scpedicini** [reply]
Good to hear - might want to mark in the dropdown with a strikethrough or something in the meantime. I spent a good 15 minutes trying to figure out if something was wrong with my custom prompt before finding this comment.

**Reply by danny-avila** [reply]
Sorry about that. Shouldn't be necessary though as it goes live later today or tomorrow if I don't get to it!

**Comment by danny-avila** [comment]
Vision API is now supported :)
